test/unit/tsCheckAndRun.test.js

```diff
@@ -0,0 +1,47 @@
+const {join} = require('path')
+const { describe, test } = require('@jest/globals')
+const { runTyperAndTsCheck } = require('../util')
+const cds = require('@sap/cds')
+
+describe('Generate, TS Check, Run ', () => {
+
+    test.each([
+        'actions',
+        // 'arrayof',
+        // 'aspects',
+        // 'autoexpose',
+        // 'bookshoplet',
+        // 'builtins',
+        // 'count',
+        // 'delimident',
+        // 'draft',
+        // 'enums',
+        // 'events',
+        // 'excluding',
+        // 'foreignkeys',
+        // 'hana',
+        // 'inflection',
+        // 'inheritance',
+        // 'inline',
+        // 'notnull',
+        // 'projection',
+        // 'references',
+        // 'scoped',
+        // 'type',
+        // 'typeof',
+        // 'views',
```

Comment:
I see that you are currently just testing the waters with `actions`, but still wanted to note that this can be [resolved dynamically](https://github.com/cap-js/cds-typer/blob/main/test/smoke/transpilation.test.js#L8) for all existing model directories

Reply:
Done